etc/kayobe/ansible/growroot.yml

```diff
@@ -27,57 +27,60 @@
     growroot_vg: "rootvg"
     # Don't assume facts are present.
     os_family: "{{ ansible_facts.os_family | default('Debian' if os_distribution == 'ubuntu' else 'RedHat') }}"
+    # Ignore LVM check
+    growroot_ignore_lvm_check: false
 
   tasks:
-    - name: Check if growpart is installed
+    - name: Check LVM status
       shell:
-        cmd: type growpart
```

Comment:
Hmm, looking at this again we've lost the growpart check. I think that was trying to avoid package installation during early system bootstrapping where possible, since we might not have network and proxy configuration in place.

Reply:
Ah that makes sense. I removed it initially as i thought it's purpose was fulfilled by the install task.